bleachbit.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
@Zopolis4 
This commit has a mix of `#!/usr/bin/env python3` and #!/usr/bin/env python`. Let's use the former because only Python3 is supported.